MailSim/MailSim/ExecuteSequence.cs

```diff
@@ -742,9 +742,17 @@ private bool EventMonitor(MailSimOperationsEventMonitor operation)
                 MailFolder folder = olMailStore.GetDefaultFolder(operation.Folder);
                 if (folder == null)
                 {
-                    Log.Out(Log.Severity.Error, operation.OperationName, "Unable to retrieve folder {0}",
-                     operation.Folder);
-                    return false;
+                    // might be a custom folder under the Inbox (right now we'll only support folders under the Inbox)
+                    MailFolder inboxFolder = olMailStore.GetDefaultFolder("olFolderInbox");
```

Comment:
This is definitely a simple first stab.  I was thinking Inbox subfolders was the most likely  scenario but a more general approach might be better for more flexible testing scenarios, as Alex mentioned.